Marlin/Configuration.h

```diff
@@ -639,6 +643,10 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
  #define ENCODER_STEPS_PER_MENU_ITEM 1
 #endif
 
+#if defined (PANEL_ONE)
+ #define SDSUPPORT
+ #define ULTIMAKERCONTROLLER
+#endif
```

Comment:
Sweet! If you wouldn't mind adding these lines to the other `Configuration.h` files (in example_configurations) we can merge this forthwith!